server/commands/write_authzmodel.go

```diff
@@ -29,14 +31,23 @@ func NewWriteAuthorizationModelCommand(
 
 // Execute the command using the supplied request.
 func (w *WriteAuthorizationModelCommand) Execute(ctx context.Context, req *openfgapb.WriteAuthorizationModelRequest) (*openfgapb.WriteAuthorizationModelResponse, error) {
+	schemaVersion := req.GetSchemaVersion()
+	if schemaVersion == "" {
+		schemaVersion = "1.0"
+	}
+
+	if _, ok := supportedSchemaVersions[schemaVersion]; !ok {
+		return nil, serverErrors.UnsupportedSchemaVersion
+	}
+
 	typeDefinitions := req.GetTypeDefinitions().GetTypeDefinitions()
 
 	// Until this is solved: https://github.com/envoyproxy/protoc-gen-validate/issues/74
 	if len(typeDefinitions) > w.backend.MaxTypesInTypeDefinition() {
 		return nil, serverErrors.ExceededEntityLimit("type definitions in an authorization model", w.backend.MaxTypesInTypeDefinition())
 	}
 
-	if err := validateAuthorizationModel(typeDefinitions); err != nil {
+	if err := validateAuthorizationModel(req.GetSchemaVersion(), typeDefinitions); err != nil {
```

Comment:
`schemaVersion` is already defined above.

Reply:
Oops. Thank you! Fixed.